doc/source/conf.py

```diff
@@ -73,15 +91,21 @@
 # Intersphinx mapping
 intersphinx_mapping = {
     "python": ("https://docs.python.org/dev", None),
+    "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
+    "numpy": ("https://numpy.org/devdocs", None),
+    "matplotlib": ("https://matplotlib.org/stable", None),
+    "pyvista": ("https://docs.pyvista.org/", None),
     # kept here as an example
-    # "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
-    # "numpy": ("https://numpy.org/devdocs", None),
-    # "matplotlib": ("https://matplotlib.org/stable", None),
     # "pandas": ("https://pandas.pydata.org/pandas-docs/stable", None),
-    # "pyvista": ("https://docs.pyvista.org/", None),
     # "grpc": ("https://grpc.github.io/grpc/python/", None),
 }
 
+# suppress annoying matplotlib bug
+warnings.filterwarnings("ignore", category=UserWarning)
+
+suppress_warnings = ["label.*"]
+# supress_warnings = ["ref.option"]
+
```

Comment:
Maybe we can add here the annoying VTK warnings?

Reply:
I tried to do it that way but I didn't succeed